src/main/java/core/basesyntax/EmployeeService.java

```diff
@@ -1,10 +1,24 @@
 package core.basesyntax;
 
+import java.util.Comparator;
 import java.util.List;
 import java.util.Set;
+import java.util.TreeSet;
 
 public class EmployeeService {
     public Set<Employee> getEmployByOrder(List<Employee> employees) {
-        return null;
+        Set<Employee> employeeSet = new TreeSet<>(employeeComparator());
```

Comment:
When creating a new TreeSet with a custom comparator, it is more robust to specify the comparator directly in the TreeSet constructor, instead of calling a separate method. This ensures that the comparator's implementation is clear and visible right where the TreeSet is instantiated.